CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.0]
+### Changed
+- Fix: changes in subscription middleware to not throw error if block for a block number is not found ([#89](https://github.com/MetaMask/eth-json-rpc-filters/pull/89))
```

Comment:
Our breaking change entries conventionally start with `**BREAKING:** `

Also maybe better to describe this as "filter middleware", as they can be used independently of the subscriptions as well. Here is a suggested rewording:
```suggestion
- **BREAKING:** Block filter middleware will not throw an error if a block is not found ([#89](https://github.com/MetaMask/eth-json-rpc-filters/pull/89))
```